src/test/java/seedu/todo/logic/commands/ClearCommandTest.java

```diff
@@ -0,0 +1,54 @@
+package seedu.todo.logic.commands;
+
+import java.time.LocalDateTime;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.todo.commons.exceptions.ValidationException;
+import seedu.todo.model.property.TaskViewFilter;
+import seedu.todo.model.task.ImmutableTask;
+
+public class ClearCommandTest extends CommandTest {
+    
+    private ImmutableTask task1, task2, task3;
+
+    @Before
+    public void setUp() throws Exception {
+        task1 = model.add("Hello", task -> {task.setCompleted(true); 
+                                    task.setEndTime(LocalDateTime.now().plusHours(2));
+                                    task.setPinned(true);
+                                    } );
```

Comment:
This is not the correct way to indent lambdas. Treat the `{ }` like any other braces - 

``` java
model.add("Hello", task -> {
    /// ... 
});
```